lib/fog/openstack.rb

```diff
@@ -59,6 +59,10 @@ module Metric
     autoload :OpenStack, File.expand_path('../metric/openstack', __FILE__)
   end
 
+  module Event
```

Comment:
Yep, created a new commit (part of this PR) updating also some other service names.